module/tools.js

```diff
@@ -158,7 +158,7 @@ export function calculateWound(damage, size) {
     if(typeOfWoundsBySize === undefined) return false;
```

Comment:
I cannot comment on unedited code, but line 154 above should be :
if(damage <**=** 0) {

Reply:
Done!